dojo/tools/scout_suite/parser.py

```diff
@@ -0,0 +1,136 @@
+            test_description = "%s\n* **Checked Items:** %s\n" % (test_description, items["checked_items"])
+            test_description = "%s* **Flagged Items:** %s\n" % (test_description, items["flagged_items"])
+            test_description = "%s* **Max Level:** %s\n" % (test_description, items["max_level"])
+            test_description = "%s* **Resource Count:** %s\n" % (test_description, items["resources_count"])
+            test_description = "%s* **Rules Count:** %s\n\n" % (test_description, items["rules_count"])
+        test.description = test_description
+        test.save()
+
+        scout2_findings = []
+
+        # Configured Services
+        for service in list(data["services"].items()):
+            for service_item in service:
+                if "findings" in service_item:
+                    for name, finding in list(service_item["findings"].items()):
+                        if finding["items"]:
+                            description_text = ""
+                            for name in finding["items"]:
+                                description_text = description_text + "**Location:** " + name + "\n\n---\n"
+                                description_text = description_text + "\n"
+                                key = name.split('.')
+                                i = 1
+                                lookup = service_item
+                                while i < len(key):
+                                    if key[i] in lookup:
+                                        if (type(lookup[key[i]]) is dict):
+                                            lookup = lookup[key[i]]
+                                            if (key[i - 1] == "security_groups" or key[i - 1] == "PolicyDocument"):
+                                                break
+                                    i = i + 1
+
+                                self.recursive_print(lookup)
+                                description_text = description_text + self.item_data
+                                self.item_data = ""
+
+                            mobsf_item = {
+                                "category": "Mobile Permissions",
+                                "title": finding["description"],
+                                "severity": finding["level"],
+                                "description": description_text
+                            }
+                            scout2_findings.append(mobsf_item)
+
+        for scout2_finding in scout2_findings:
+            title = strip_tags(scout2_finding["title"])
+            sev = self.getCriticalityRating(scout2_finding["severity"])
+            description = scout2_finding["description"]
+            dupe_key = sev + title
+            if dupe_key in dupes:
+                find = dupes[dupe_key]
+                if description is not None:
+                    find.description += description
+            else:
+                find = Finding(title=Truncator(title).words(6),
```

Comment:
If you cut it, could be good to add "..." to the string

Reply:
Truncator does it automatically.
https://docs.djangoproject.com/en/1.8/_modules/django/utils/text/

Reply:
oh